index.html

```diff
@@ -4,21 +4,25 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
     <link rel="stylesheet" href="styles.css">
     <title>Andrew's portfolio</title>
 </head>
 <body>
   <div class="container">
-      <div class="box1">MY Logo</div>
+      <div class="logo">
+          <div class="box1">MY Logo</div>
+          <div ><i class="material-icons">menu</i></div>
+        </div>
       <div class="box2">I'm Andrew Glade to see you!</div>
       <div class="box3">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</div>
```

Comment:
- [ ] The paragraph (e.g., "I'm a software ...") is a `p` tag, Kindly, refactor that .. that will improve your project to look more semantic 😄